snakes_cafe.py

```diff
@@ -0,0 +1,121 @@
+"""
+Lab: Class 1 - Intro to Python
+Overview
+Today you’ll begin working on a command line utility which will mimic the functionality of a point
+of sale restaurant system using your basic Python tools and understanding of the basics of the language.
+"""
+import sys
+
+"""
+Build a restaurant point of sale/order system
+Feature Tasks and Requirements
+* When run, the program should print an intro message and the menu for the restaurant
+* The restaurant’s menu should include appetizers, entrees, desserts, and beverages.
+* The program should prompt the user for an order
+* When a user enters an item, the program should print an acknowledgment of their input
+* The program should tell the user how to exit
+* The program’s content should match included sample exactly
+    * Actually, there’s one tiny spot that should be different - see if you can spot it.
+    * The > character represents user input line and should be printed out with a trailing space.
+
+pseudo code:
+1) create functions for the different functionality you need.
+2) build logical control flow that calls the functions as needed.
+3) could build this as a while loop with while true, < would prefer to find another method of handing this flow
+
+"""
+appetizers = ['Wings', 'Cookies', 'Spring Rolls']
+entrees = ['Salmon', 'Steak', 'Meat Tornado', 'A Literal Garden']
+desserts = ['Ice Cream', 'Cake', 'Pie']
+drinks = ['Coffee', 'Tea', 'Unicorn Tears']
+menu_items = ['Wings', 'Cookies', 'Spring Rolls', 'Salmon',
+              'Steak', 'Meat Tornado', 'A Literal Garden',
+              'Ice Cream', 'Cake', 'Pie',
+              'Coffee', 'Tea', 'Unicorn Tears']
+
+
+def display_menu_items(menu_list):
+    # pass any of the above lists into display_menu_items
+    # to print that menu section to screen.
+    for item in menu_list:
+        print(item)
+    print("\n")
+
+
+def handle_order():  # this needs logic to check the order_list for repeats and display the number.
+    user_order = input('> ').capitalize()
+    if user_order in menu_items:
+        return user_order
+    elif user_order == 'Quit':
+        return user_order
+    else:
+        print("That item is not on the menu, please order from the menu")
+        handle_order()
```

Comment:
You have a bug here. If an order of `"potato"` comes along, the second instance of `handle_order()` executes, but the original instance isn't given a return a returns None inside of `snakes_cafe()`.

As a result, following `"potato"` with `"Salmon"` completes the second `handle_order()`, but actually sets `order` on line 66 to `None`. This gets added to the `oder_dict` dictionary of orders and starts tracking `None` as part of the order.